manifests/minimize_access.pp

```diff
@@ -32,7 +32,9 @@
   }
   # from which folders to remove public access
   $folders = [
+    '/usr/local/games',
     '/usr/local/sbin',
+    '/usr/local/bin',
```

Comment:
This folder collides with a resource definition in my Puppet Enterprise 2018.1 (/opt/puppetlabs/puppet/modules/puppet_enterprise/manifests/symlinks.pp, line: 31)

Reply:
I do not have access to the PE code. Do you have a suggestion on how to solve this? I can wrap it in an `if defined` clause, if you like? The thing is, for the CIS Benchmark, this should be set correctly so the class should check it, IMHO.

Reply:
Hmm, good question ... maybe you could define those resources with [ensure_resources](https://forge.puppet.com/puppetlabs/stdlib#ensure_resource)?